File: LABS C++/simple.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string estadotarjeta;

    // Se le pide el estado de cuenta al usuario
    cout << "Ingrese el estado de su cuenta (activa/inactiva): ";
    cin >> estadotarjeta;

    // Verificar si la tarjeta está activa
    if (estadotarjeta == "activa") {
        cout << "Puedes realizar prestamos." << endl;

    } if (estadotarjeta == "inactiva") {
        cout << "No puedes realizar prestamos." << endl;
    }

    return 0;
}

File: LABS JAVA/practica/src/simple.java
import java.util.Scanner;

public class simple {
    public static void main(String[] args) {
      Scanner scanner = new Scanner(System.in);

// Se solicita al usuario su estado de cunta
        System.out.print("Introduce el estado de tu cuenta (activa o no activa): ");
        String estadoTarjeta = scanner.nextLine();


    // se determina si puede o no puede realizar prestamos
        if (estadoTarjeta.equalsIgnoreCase("activa")) {
          System.out.println("Puedes realizar prestamos.");

      }  if (estadoTarjeta.equalsIgnoreCase("no activa")) {
          System.out.println("Los prestamos no etan permitidos");
      
      }
      
      scanner.close();
  } 
}